composer.json

```diff
@@ -30,7 +30,7 @@
     "cultuurnet/search-v3": "dev-master",
     "cultuurnet/silex-amqp": "~0.1",
     "cultuurnet/udb2-domain-events": "~0.1",
-    "cultuurnet/udb3": "~0.1",
+    "cultuurnet/udb3": "dev-move-event-related-code-to-udb3-silex as 0.1",
```

Comment:
Should we switch this back?

Reply:
Only after we merge the other PR in my opinion. I changed it here to prove the CI would succeed after the code was removed from the dependency.